tools/ReleaseTool/Common.cs

```diff
@@ -40,5 +44,21 @@ public static string AppendDirectorySeparator(string originalPath)
 
             return originalPath + directorySeparator;
         }
+
+        public class WorkingDirectoryScope : IDisposable
```

Comment:
Cute - split it out into its own file?

Reply:
Sure - I was actually a little surprised there wasn't something like this in the standard library, as its just `pushd` and `popd`.